website/source/docs/agent/caching/index.html.md

```diff
@@ -13,55 +13,77 @@ description: |-
 
 Vault Agent Caching allows client-side caching of responses containing newly
 created tokens and responses containing leased secrets generated off of these
-newly created tokens. The renewals of the secrets that are cached are also
+newly created tokens. The renewals of the cached tokens and leases are also
 managed by the agent.
 
 ## Caching and Renewals
 
-Caching and renewals are managed by the agent only under these scenarios.
+Caching and renewals are managed by the agent only under these specific scenarios.
 
 1. Token creation requests are made through the agent and not directly to the
-   Vault server. Login endpoints from various auth methods also fall under this
-   category along with token creation endpoints of the token auth method.
+   Vault server. This means that any login operations performed using various
+   auth methods and invoking the token creation endpoints of the token auth
+   method via the agent will result in the authentication response getting
+   cached by the agent. Note that the child tokens will only be cached by the
+   agent if the parent token is already being managed by the agent (the
+   exception to this is when an agent unmanaged token is used to create an
+   orphan token; the orphan token will be cached and managed by the agent).
 
 2. Leased secret creation requests are made through the agent using tokens that
-   are already managed by the agent.
+   are already managed by the agent. This means that any dynamic credentials
+   that are issued using the tokens managed by the agent, will be cached by the
+   agent and its renewals are taken care of.
 
 ## Using Auto-Auth Token
 
-Clients do not need to provide a Vault token along with the proxied request if
-the [auto-auth](/docs/autoauth/index.html) feature is enabled. This feature is
-enabled by setting the `use_auto_auth_token` (see below) configuration field.
-However, even when enabled, if requests that reach the agent already have a
-token attached on them, the attached token will be put to use instead of the
-auto-auth token.
+Vault Agent allows for easy authentication to Vault in a wide variety of
+environments using [Auto-Auth](/docs/agent/autoauth/index.html). By setting the
+`use_auto_auth_token` (see below) configuration, clients will not be required
+to provide a Vault token to the requests made to the agent. When this
+configuration is set, if the request doesn't already bear a token, then the
+auto-auth token will be used to forward the request to the Vault server. This
+configuration will be overridden if the request already has a token attached.
+In this case, the token present in the request will be used to forward the
+request to the Vault server.
 
 -> **Note:** In Vault 1.1-beta, if the request doesn't already contain a Vault
```

Comment:
This should be enabled for beta2. We can either remove this or update it to mention that this is the case for beta1 specifically.

Reply:
I pushed some more changes before looking at this comment. Please check if this is addressed.